QuizUp/actions/upload_to_s3.rb

```diff
@@ -0,0 +1,44 @@
+module Fastlane
+  module Actions
+    class UploadToS3Action < Action
+      def self.run(options)
+
+        s3 = Aws::S3::Resource.new
+        file_path = options[:ipa]
+        obj = s3.bucket(ENV['SECRET-PROJECT-BUCKET-NAME']).object(File.basename(file_path))
```

Comment:
@KrauseFx sorry for taking so long to fix this small thing, I forgot I had this PR open :(